dev/bin/minikube-start.sh

```diff
@@ -48,7 +50,11 @@ if [ "${disable_extra_components}" != 'true' ]; then
   sleep 5
   kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/kube-prometheus/v0.9.0/manifests/setup/prometheus-operator-0servicemonitorCustomResourceDefinition.yaml
   . "${SCRIPT_DIR_PATH}/knative-installer.sh"
-  yes | istioctl manifest apply --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  istioctl manifest apply --skip-confirmation --set profile=default --set values.gateways.istio-ingressgateway.type="ClusterIP"
+  sleep 5
+  # Camel-K installation
+  kubectl create namespace camel-k --dry-run=client -o yaml | kubectl apply -f -
+  kamel install --runtime-version 1.10.1 -n camel-k --force -w
```

Comment:
Can you move this code to a script so that we can reuse it also for CI? Like we did for [knative](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/a830b6d610455491304009628dd1c44ce48bf915/dev/bin/minikube-start.sh#L52)

Reply:
+1 to this comment

Reply:
done :+1: